Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -111,6 +111,10 @@ public class XPCServer {
     public static func forThisXPCService() throws -> XPCServer {
         try XPCServiceServer._forThisXPCService()
     }
+
+    public static func makeAnonymousService() -> XPCServer & NonBlockingStartable {
+        XPCAnonymousServer()
```

Comment:
`XPCAnonymousServer` doesn't conform to `XPCAnonymousServer` so this won't compile.